ssh/tailssh/incubator.go

```diff
@@ -310,15 +310,22 @@ func (ss *sshSession) launchProcess() error {
 	if err != nil {
 		return err
 	}
-	go resizeWindow(pty, winCh)
-	ss.stdout = pty // no stderr for a pty
-	ss.stdin = pty
+	// We need to handle closing the the stdin and stdout separately from each other;
```

Comment:
typo the the. I'd say:

"We need to be able to close stdin and stdout separately later so make a dup."